apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsFeaturesContext.tsx

```diff
@@ -5,14 +5,14 @@ import React, { createContext, useCallback, useState, useContext } from 'react'
 
 // Specify all data needed for specific feature
 type MapGenerationFeatureContext = {
-    feature: B2BAppGlobalFeature.PropertyMapGeneration,
-    propertyId: Scalars['ID']
+    feature: B2BAppGlobalFeature.PropertyMapGeneration
+    propertyId: Scalars['ID']['input']
 }
 
 type AttachCallRecordToTicketFeatureContext = {
-    feature: B2BAppGlobalFeature.AttachCallRecordToTicket,
-    ticketId: Scalars['ID'],
-    ticketOrganizationId: Scalars['ID'],
+    feature: B2BAppGlobalFeature.AttachCallRecordToTicket
+    ticketId: Scalars['ID']['input']
+    ticketOrganizationId: Scalars['ID']['input']
```

Comment:
what is this about?

Reply:
Take a look at the updated "schema.ts". Scalars are now generated a little differently